Rocket.Chat/Views/Cells/Preferences/ReusableViewText.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ReusableViewText.swift
+//  Rocket.Chat
+//
+//  Created by Rafael Kellermann Streit on 07/03/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+final class ReusableViewText: UICollectionReusableView {
+
+    static let identifier = "ReusableViewTextIdentifier"
```

Comment:
Nothing specifically related to this PR, just a note: what if we start using `String(describing: Self.self)` for identifiers like this one in the future?

Reply:
Sounds good to me... could even be part of a Generic? 🤔

Reply:
Cool. Yes, it could :D